src/app/screens/connectors/ConnectAlby/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import Button from "~/app/components/Button";
+import msg from "~/common/lib/msg";
+import { WebLNNode } from "~/extension/background-script/connectors/connector.interface";
+
+type Props = {
+  fromWelcome?: boolean;
+};
+
+export default function ConnectAlby({ fromWelcome }: Props) {
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate();
+  const { t: tCommon } = useTranslation("common");
+  async function connectAlby() {
+    setLoading(true);
+    const name = "Alby";
+    const initialAccount = {
+      name,
+      config: {},
+      connector: "alby",
+    };
+
+    try {
+      const validation = await msg.request("validateAccount", initialAccount);
+      if (validation.valid) {
+        if (!validation.oAuthToken) {
+          throw new Error("No oAuthToken returned");
+        }
+
+        const account = {
+          ...initialAccount,
+          name: (validation.info as { data: WebLNNode }).data.alias,
```

Comment:
use email, fallback to username part of lightning address, finally fallback to "getalby.com"

Reply:
can we somehow use some less fancy syntax here?

Reply:
Changed to 
`name: accountInfo.email || accountInfo.lightning_address || "getalby.com",`

![image](https://github.com/getAlby/lightning-browser-extension/assets/33993199/1a6f62e9-fdde-4cdd-9803-0313f9a5877f)